packages/preact/src/hooks/useSwatches/useSwatches.ts

```diff
@@ -62,13 +63,14 @@ import { applySelectionStates } from "./applySelectionStates"
 export function useSwatches(skus: SearchProductSku[] = [], fields: string[] = []) {
   const [selectedOptions, setSelectedOptions] = useState<Record<string, string>>({})
 
-  const aggregatedSwatches = useMemo(() => {
-    return aggregateSwatches(skus, fields)
-  }, [skus, fields])
-
   const swatches = useMemo(() => {
```

Comment:
with this kind of useMemo usage the following logic will be recomputed on selection state change
* aggregation
* sorting
* applySelectionStates

is that intentional?